src/ability/conditions/voyage-teams.ability.ts

```diff
@@ -17,6 +18,43 @@ export const manageOwnVoyageTeamWithIdParam = (
     }
 };
 
+export const userCanChangeCategory = async (
+    categoryId: number,
+    user: UserReq,
+    teamId: number | undefined = undefined,
+) => {
+    if (user.roles?.includes("admin")) return;
+
+    let match;
```

Comment:
https://github.com/chingu-x/chingu-dashboard-be/blob/539c7d6f35fd2408a2246fb1c41c507950b9f97d/src/ability/conditions/voyage-teams.ability.ts#L28-L35
Since `match` is assigned inside the following `try` block,  I declared it outside of the block so it will still be in scope later